ts/receiver/types.ts

```diff
@@ -0,0 +1,101 @@
+
+enum MessageModelType {
+    INCOMING = "incoming",
+    OUTGOING = "outgoing",
+    FRIEND_REQUEST = "friend-request"
+}
+
+export enum EndSessionType {
+    DONE = 'done',
+    ONGOING = 'ongoing',
+}
+
+interface MessageAttributes {
```

Comment:
this should probably be put in `messages.d.ts` next to `messages.js` file in models

Reply:
done